src/main/java/com/wolox/training/models/Book.java

```diff
@@ -0,0 +1,136 @@
+package com.wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity(name = "book")
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column(name = "genre")
+    private String genre;
+
+    @Column(name = "author", nullable = false)
```

Comment:
Add `@NotNull` to every non-null property, in order to validate the existence not only at database level

Reply:
Solved, add @NotNull property in the fields